src/client/pages/UserPage/components/UserProfile.tsx

```diff
@@ -5,6 +5,8 @@ import React from 'react';
 import { Field, reduxForm, InjectedFormProps, WrappedFieldProps } from 'redux-form';
 import { CustomFieldProps } from '~/components/Fields';
 import FileField from '~/components/FileField';
+import { generateQRCodeURL } from '@Shared/QRCodes';
```

Comment:
Could we please rename this method to have ProperCasing `GenerateQRCodeURL`. For exported methods it just really makes it look nicer (and you can assume it's exported)

Reply:
We don't do this anywhere else, eg: https://github.com/UCSDTESC/Check-in/blob/master/src/shared/Roles.ts if you do end up doing this rename all of them :)

Reply:
Oopsies. That's my bad :) But yeah... we should really be using ProperCase.

Reply:
I made an issue for this (#185) so I can do it later and keep it separate from this pr. for now, I left it to be consistent with everything else